draft/2024-02-14-this-week-in-rust.md

```diff
@@ -35,6 +35,8 @@ and just ask the editors to select the category.
 
 ### Project/Tooling Updates
 
+- [zbus 4.0 — release notes](https://github.com/dbus2/zbus/releases/tag/zbus-4.0.0)
+
```

Comment:
Seems the template uses the footnote links so we should too and it would be good to have a bit of description here (from previous issues, I see that's the trend).

```suggestion
- [zbus 4.0][zbus4] released. [zbus][zbus] is a pure Rust D-Bus crate. The new version brings a more ergonomic and safer API.

[zbus4]: https://github.com/dbus2/zbus/releases/tag/zbus-4.0.0
[zbus]: https://github.com/dbus2/zbus
```